easybuild/easyconfigs/j/jax/jax-0.7.0-gfbf-2025a.eb

```
@@ -0,0 +1,97 @@
+# Updated by: Thomas Hoffmann (EMBL Heidelberg)
+# Updated by: Pavel Tománek (INUITS)
+
+easyblock = 'PythonBundle'
+
+name = 'jax'
+version = '0.7.0'
+
+homepage = 'https://jax.readthedocs.io/'
+description = """Composable transformations of Python+NumPy programs:
+differentiate, vectorize, JIT to GPU/TPU, and more"""
+
+toolchain = {'name': 'gfbf', 'version': '2025a'}
+
+builddependencies = [
+    ('Bazel', '7.4.1', '-Java-21'),
+]
+
+dependencies = [
+    ('Python', '3.13.1'),
+    ('SciPy-bundle', '2025.06'),
+    ('absl-py', '2.3.1'),
+    ('flatbuffers-python', '25.2.10'),
+    ('ml_dtypes', '0.5.1'),
+    ('hypothesis', '6.133.2'),
+    ('zlib', '1.3.1'),
+]
+
+# downloading xla  tarball to avoid that Bazel downloads it during the build
+# note: following commits *must* be the exact same onces used upstream
+# XLA_COMMIT from jax-jaxlib: third_party/xla/workspace.bzl
+_xla_commit = '3d25fed5571304e446903bc00e4f457b2b0f73dc'
+
+# Use sources downloaded by EasyBuild
+_jaxlib_buildopts = '--bazel_options="--distdir=%(builddir)s/archives" '
+# create wheels for jaxlib and install it
+_jaxlib_buildopts += '--wheels=jaxlib '
+# use GCC instead of default Clang
+_jaxlib_buildopts += '--use_clang=false --gcc_path=$CC '
+# fix jaxlib version
+_jaxlib_buildopts += "--bazel_options='--action_env=JAXLIB_RELEASE' "
+
+components = [
+    ('jaxlib', version, {
+        'sources': [
+            {
+                'source_urls': ['https://github.com/google/jax/archive/'],
+                'filename': 'jax-v%(version)s.tar.gz',
+            },
+            {
+                'source_urls': ['https://github.com/openxla/xla/archive'],
+                'download_filename': f'{_xla_commit}.tar.gz',
+                'filename': f'xla-{_xla_commit[:8]}.tar.gz',
+                'extract_cmd': 'mkdir -p %(builddir)s/archives && cp %s %(builddir)s/archives',
+            },
+        ],
+        'patches': ['jax-0.7.0_fix-mosaic.patch'],
+        'checksums': [
+            {'jax-v0.7.0.tar.gz':
+             '518966801e4402667e77915c2dc7cf1a178a80e22ff253204a837f207a87fcde'},
+            {'xla-3d25fed5.tar.gz':
+             '9efd7d303edab24fd8552d602045722f462870d17b888fa607e9b7143b9e0515'},
+            {'jax-0.7.0_fix-mosaic.patch':
+             '8ce4bcfd1cc6a0e42288c58ee28af5d29363ca72dbde3436ee60c336f1efc575'}
+        ],
+        'start_dir': f'jax-jax-v{version}',
+        # fix jaxlib version - removes .dev suffix
+        'prebuildopts': 'export JAXLIB_RELEASE=%(version)s && ',
+        'buildopts': _jaxlib_buildopts,
+    }),
+]
+
+# BackendsTest is failing on systems with GPU detected when cpu version is built
+_failing_backend_test = 'tests/api_test.py::BackendsTest::test_no_backend_warning_on_cpu_if_platform_specified'
+# Failing after fix jaxlib version by JAXLIB_RELEASE
+_failing_version_tests = 'tests/version_test.py'
+_runtest_cmd = f"pytest -n %(parallel)s tests --deselect={_failing_backend_test} --deselect={_failing_version_tests}"
```

Maybe we need to run fewer tests in parallel here, since they're often quite memory hungry:
```suggestion
_runtest_cmd = f"pytest -n $((%(parallel)s / 4)) tests --deselect={_failing_backend_test} --deselect={_failing_version_tests}"
```

or even `/ 8`, worst case we're not running tests in parallel (which would be slow, but if that makes them not fail...)